### PR TITLE
add upgrading to v1.2 doc

### LINKED ISSUE #2028: add "Upgrading to 1.2" page with detailed instructions
Right now in our docs we have an "Upgrading to 1.1" page that's listed in the TOC under "Install" - https://velero.io/docs/master/upgrade-to-1.1/. Let's replace this with an "Upgrading to 1.2" page.

- Image is now `velero/velero` (update for both deployment and daemonset if using restic)
- Need to add separate plugin image to deployment if using AWS/Azure/GCP
- anything else?

cc @carlisia @prydonius @nrb

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <krisss@vmware.com>

Closes #2028

- [x] test
- [x] fill in `velero install crds` command per #2034

### CODE REVIEW

**Comment by skriss** [comment]
@carlisia @nrb @prydonius I fixed one small bug (see latest commit) but otherwise this worked as expected, including with the CRD upgrade command. Ready for review!

File: site/docs/master/upgrade-to-1.2.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+## Prerequisites
+- Velero [v1.1][0] or [v1.0][1] installed.
+
+_Note: if you're upgrading from v1.0, follow the [upgrading to v1.1][2] instructions first._
+
+## Instructions
+
+1. Install the Velero v1.2 command-line interface (CLI) by following the [instructions here][3].
+
+   Verify that you've properly installed it by running:
```
Comment (carlisia) [inline site/docs/master/upgrade-to-1.2.md]: ❤️

File: site/docs/master/upgrade-to-1.2.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Scale down the existing Velero deployment:
+
+   ```bash
+   kubectl scale deployment/velero \
+       --namespace velero \
+       --replicas 0
+   ```
+
+1. Update the container image used by the Velero deployment and optionally, the restic daemon set:
```
Comment (carlisia) [inline site/docs/master/upgrade-to-1.2.md]:
```suggestion
1. Update the container image used by the Velero deployment and, optionally, the restic daemon set:
```

File: site/docs/master/upgrade-to-1.2.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+   kubectl set image daemonset/restic \
+       restic=velero/velero:v1.2.0 \
+       --namespace velero
+   ```
+
+1. If using AWS, Azure, or GCP, add the respective plugin to your Velero deployment:
+
+   For AWS:
+   ```bash
+   velero plugin add velero/velero-plugin-for-aws:v1.0.0
```
Comment (carlisia) [inline site/docs/master/upgrade-to-1.2.md]: Shouldn't these be v1.2.0?

Comment (carlisia) [reply site/docs/master/upgrade-to-1.2.md]: Oh nvm! We decided not to mirror the version numbers 🤦‍♀

**Pull request merged**